code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -83,17 +83,18 @@
 			new /obj/item/encryptionkey/syndicate(src) // 2TC
 			return
 
-		if("hacker") // 30TC + one 0TC
+		if("hacker") // 37TC + two 0TC
 			new /obj/item/aiModule/syndicate(src) // 12TC
 			new /obj/item/card/emag(src) // 6TC
 			new /obj/item/encryptionkey/syndicate(src) // 2TC
 			new /obj/item/encryptionkey/binary(src) // 5TC
 			new /obj/item/aiModule/toyAI(src) // 0TC
-			new /obj/item/multitool/ai_detect(src) // 1TC
-			new /obj/item/storage/box/syndie_kit/c4 // 4TC
+			new /obj/item/clothing/glasses/chameleon/thermal(src) // 6TC
+			new /obj/item/storage/belt/military/traitor/hacker(src) // 5TC
+			new /obj/item/clothing/gloves/color/yellow(src) // 0TC
```

Comment:
Maybe combat gloves instead of insulateds? I dont think traitors get insulateds anywhere else.

Reply:
Insulated are pretty necessary if you're going to be hackerman. Are there any gloves that are insulated and don't stand out as much? Maybe create a special black insulated glove option just for the kit.

Reply:
Combat gloves (Used by nuke ops, the blueshield, and found in syndicate toolboxes) are insulated and fireproof.